Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings

```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "Přidat zkratku";
+"press_shortcut" = "Stiskněte klávesy";
+"keyboard_shortcut_used_by_menu_item" = "Tuto zkratku nelze použít, protože ji už využívá položka „%@“ v nabídce.";
```

Comment:
"Tuto zkratku nelze použít, protože je již využívána položkou „%@“." imho sound better in this context.